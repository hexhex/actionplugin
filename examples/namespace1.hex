#namespace("ex","http://example.org#").
a("ex:foo").
b("ex:foo").
d("http://example.org#bar").
c(X) :- a(X), b(X).
