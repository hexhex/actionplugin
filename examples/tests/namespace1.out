{c("<http://example.org#foo>"), d("http://example.org#bar"), b("<http://example.org#foo>"), a("<http://example.org#foo>")}
